### PR TITLE
Basic

### PR DESCRIPTION
**Pull request by gidden**
These are all of my basic edit suggestions.

### CODE REVIEW

**Comment by gidden** [comment]
sorry to have a slight +10 line count =(

File: fundamentals/examples.tex [hunk 1]
```diff
@@ -212,7 +211,7 @@ \subsection{Simulations}
 Modifying the input file specification to model each of the 3 fuel cycles was
 done by making simple adjustments to the commodities and trade preferences for
 each of the facilities.  The reactor was configured to request recycled \gls{MOX}
-fuel with a higher preference then new \gls{UOX} fuel. For the once through case,
+fuel with a higher preference than new \gls{UOX} fuel. For the once through case,
```
Comment (katyhuff) [inline fundamentals/examples.tex]: nice catch

File: fundamentals/impl.tex [hunk 2]
```diff
@@ -336,10 +331,10 @@ \subsubsection{Resources and Materials}
 
 \end{itemize}
 
-In order to avoid excess decay calculations, when decay
-is invoked, a material checks to see if it contains any nuclides with decay
-constants that are significant with respect to the time change since the last
-decay operation.  If none of the decay constants are significant, no decay
+
+When decay is invoked, A material checks to see if it contains any nuclides with
```
Comment (katyhuff) [inline fundamentals/impl.tex]: no capital A

**Comment by katyhuff** [comment]
Perfect! Thanks @gidden !!!!

**Pull request merged**